scripts/db_metrics.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
Are we going to need this?

Reply:
I was using it in order to select requests that had appropriate numbers of items in order to generate fixtures. Not sure where to put it. I am likely going to need it again in the future. Its handy for checking the db metrics quickly.